website/docs.md

```diff
@@ -41,3 +41,15 @@ scarb lint --fix
 
 You can also specify `--test` to perform analysis of your project's tests as well (i.e. all the Cairo code under `#[cfg(test)]` attributes).
 To learn more about available arguments, just run `scarb lint --help`.
+
+## Configuration
+
+By default, most of the lints are enabled and checked, but some of them are disabled as they are much more pedantic. You can check whether a certain lint is enabled by default in the documentation. To adjust this configuration, you can manually set those values in your `Scarb.toml` as follows:
```

Comment:
It'd help to have a list of this, so you don't have to go through all the lint pages to assess what the default profile looks like

I'd also add something that would generate it so you don't have to maintain it yourself, but rather it'd be done automatically but it's an extra that is not necessary

Reply:
It can be generated automatically. Currently the information if the lint is enabled or disabled by default is already auto generated and put inside certain doc page. Will add additional page with default profile and list all disabled lints

Reply:
Added the autogenerated default profile doc page